apps/services/handlers.py

```diff
@@ -277,3 +280,37 @@ async def read_request_details(university_id: int, user_request_id: int, user=De
         "data": response,
         "message": "Got request details"
     }
+
+
+@services_router.post("/{university_id}/count-hostel-accommodation-cost/",
+                      name="create_count_hostel_accommodation_cost ",
+                      response_model=JSENDOutSchema[CountHostelAccommodationCostOut],
+                      tags=["Admin dashboard"])
+async def count_hostel_accommodation_cost(university_id: int, data: CountHostelAccommodationCostIn):
+    """
+    **Count Hostel Accommodation Cost**
+
+    **Path**:
+    - **university_id**: user university id
+
+    **Input**:
+    - **hostel_id**: hostel id
+    - **start_date_accommodation**: start date hostel accommodation of student
+    - **end_date_accommodation**: end(last) date hostel accommodation of student
+    - **bed_place_id**: student bed place id
+
+    **Return**: summa of hostel accommodation cost, based on hostel id
+    """
+    query = select(Hostel).where(Hostel.hostel_id == data.hostel_id, Hostel.hostel_id == university_id)
```

Comment:
why do u use it?
Hostel.hostel_id == university_id
1) it's wrong
2) we don't need to select hostel using university_id